packages/website/src/components/ast/Elements.tsx

```diff
@@ -47,7 +47,7 @@ export function ComplexItem({
     if (selected && !isExpanded) {
       setIsExpanded(selected);
     }
-  }, [selection, data]);
+  }, [selection, data, level, isExpanded]);
```

Comment:
Introduced bug somewhere around AST elements: you can no longer collapse an element that's visually focused.

See recording: the first 3/4 of time is in the deploy preview, where clicking a node in the source code to expand on the right, then trying to collapse, is not allowed.

https://user-images.githubusercontent.com/3335181/192619432-ff02fbc4-6794-4f77-97ab-323f03de01c7.mov

Reply:
Thank you for pointing it out. It should be fixed on commit 507c7a59bc5e2ca3ae88f879d71b5201bbd03060.